ocaml/squeezed/lib/squeeze.ml

```diff
@@ -99,23 +99,6 @@ type host = {
 
 let make_host ~domains ~free_mem_kib = {domains; free_mem_kib}
 
```

Comment:
Could you add an interface for this module? This is very easy to do using VS code with the "ocaml platform" extension. Click the "ml/mli" button on the top right of the window while the file `ocaml/squeezed/lib/squeeze.ml` is open.

Reply:
OK, let me generate a interface.